mm2src/trezor/src/client.rs

```diff
@@ -116,11 +116,10 @@ impl<'a> TrezorSession<'a> {
         };
 
         let result_handler = ResultHandler::<()>::new(move |pong: proto_common::Success| {
-            if pong.message == Some(ping_message.clone()) {
+            if pong.message == Some(ping_message) {
                 Ok(())
             } else {
-                let error = format!("Expected '{ping_message}' PONG message, found: {:?}", pong.message);
-                MmError::err(TrezorError::Failure(OperationFailure::Other(error)))
+                MmError::err(TrezorError::PongMessageMismatch)
```

Comment:
Maybe we should show the expected/found message in the error as strings
```rust
MmError::err(TrezorError::PongMessageMismatch {
    expected: ping_message,
    found: format!("{:?}", pong.message),
})
```

Reply:
Hmm good note. I re-checked `PongMessageMismatch` and decided, it is also better to add this err in `HwRpcError`, as it was a part of `OperationFailure::Other` and client ping error is likely an rpc err, not internal.